src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostOptions.cs

```diff
@@ -29,5 +29,22 @@ internal static class RemoteHostOptions
 
         [ExportOption]
         public static readonly Option<bool> RemoteHostTest = new Option<bool>(nameof(InternalFeatureOnOffOptions), nameof(RemoteHostTest), defaultValue: false);
+
+        public const int CrashVS = 0;
+        public const int ShowInfoBarAndRunFeaturesInProcIfPossible = 1;
+        public const int ShowInfoBarAndDisableAllFeatures = 2;
+
+        // Define VS behavior on unintentional OOP shutdown such as user killing OOP process
+        //
+        // CrashVS: 
+        //      crash VS
+        // ShowInfoBarAndRunFeaturesInProcIfPossible: [default]
+        //      show info bar and let VS keep running. some features might keep running inproc based on how they are implemented.
```

Comment:
these options are for features currently officially supported in RTW. once matt or shiproom decide which route we will go, I will remove all other options and all features who want to run in OOP need to write their features in a way that knows how to handle OOP killed case.

I made these options since Matt asked me until shiproom choose one of those.